core/platform/models.py

```diff
@@ -274,7 +274,7 @@ def import_bulk_email_services(cls):
             Exception. The value of feconf.BULK_EMAIL_SERVICE_PROVIDER does not
                 correspond to a valid email_services module.
         """
-        if constants.DEV_MODE:
+        if not feconf.CAN_SEND_REQUESTS_TO_BULK_EMAIL_SERVICE_PROVIDER:
```

Comment:
Wait hang on. Why isn't this constants.DEV_MODE? If this flag is turned off in production surely we shouldn't be importing dev_mode_bulk_email_services there?

Reply:
This was to fix the e2e tests and also helps in not releasing this feature until the release scripts are updated. The e2e test runs in prod mode and so throws the 'mailchimp key not available' error.

Reply:
Done (as per later comments)

Reply:
Not sure this is still the right way to go about it, please look at how other services are selected in this file and try to do something similar.

Reply:
Done, used just emulator mode. Though, this would require the release script to be updated before the release that thsi PR would go in.

Reply:
Oh -- I didn't realize. Could you please explain what needs to be changed there (and make sure that that change happens)?

Reply:
Just the changes required to populate the api key and other mailchimp related parameters in feconf (API key, Username, List ID and Secret).

Reply:
Ah ok. Yeah that should be fine, could you please kick off a separate PR for that?